packages/components/Notification/src/__tests__/__snapshots__/Notification.test.tsx.snap

```diff
@@ -78,15 +78,15 @@ exports[`Notification component tests Notification default 1`] = `
         "flexWrap": undefined,
         "justifyContent": "space-between",
         "minHeight": 52,
-        "padding": 16,
+        "padding": "16px",
         "paddingBottom": undefined,
         "paddingEnd": undefined,
         "paddingHorizontal": undefined,
         "paddingLeft": undefined,
         "paddingRight": undefined,
         "paddingStart": undefined,
         "paddingTop": undefined,
-        "paddingVertical": 12,
+        "paddingVertical": undefined,
```

Comment:
Not blocking, just a comment that this is unfortunate. Maybe this is a case where we should force jest to run on ios.

Reply:
This is due to the fact that our jest tests use the default theme instead of the iOS theme, so this is a separate issue.